Aptoide/appcoins-wallet-android
# [MOB-719] Add skeletons as loadings to payments

**What does this PR do?**

Adds skeletons to the following screen: PaymentMethodsFragment, MergedAppCoinsFragment, TopUpFragment
Several refactor had to be done to the PaymentMethodsFragment, so try to check every flow and see if there's something wrong like the skeletons appearing when they shouldn't or not disappearing.
**Note:** This adds a new library for the shimmerEffect

**Database changed?**

No

**Where should the reviewer start?**

PaymentMethodsFragment

**How should this be manually tested?**

Several refactor had to be done to the PaymentMethodsFragment, so try to check every flow and see if there's something wrong like the skeletons appearing when they shouldn't or not disappearing.
Error view display

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/MOB-719

**Questions:**

  This adds a new lib for the Shimmer effect




**Code Review Checklist**
